docs/plugins-list.md

```diff
@@ -20,28 +19,33 @@ sidebar_label: Plugins List
 - [throw-expressions](plugin-proposal-throw-expressions.md)
 - [record-and-tuple](plugin-proposal-record-and-tuple.md)
 
+#### Syntax Only
```

Comment:
I think they can be placed at the last section. They are only required if you are using an older `@babel/core`, we may want to say that here, too.